src/amo/noscript.css

```diff
@@ -0,0 +1,6 @@
+.InstallButton .InstallButton-button {
+  display: inline-block !important;
```

Comment:
Can we have comments next to all `!important` declarations? I find it really helps when doing (inevitable) CSS cleanup and prevents someone trying to be clever and remove the `!important`.